GearRent/ViewControllers/ListingReservationsViewController.m

```diff
@@ -0,0 +1,63 @@
+//
+//  ListingReservationsViewController.m
+//  GearRent
+//
+//  Created by Edwin Delgado on 7/14/22.
+//
+
+#import "ListingReservationsViewController.h"
+#import "../Models/Item.h"
+#import "../Models/Reservation.h"
+#import "../Models/TimeInterval.h"
+#import "Parse/Parse.h"
+#import "ReservationTableViewCell.h"
+
+@interface ListingReservationsViewController () <UITableViewDelegate, UITableViewDataSource>
+@property (weak, nonatomic) IBOutlet UITableView *tableView;
+@property (strong, nonatomic) NSArray *reservations;
+
+- (IBAction)didTapBack:(id)sender;
+
+@end
+
+@implementation ListingReservationsViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    self.tableView.delegate = self;
+    self.tableView.dataSource = self;
+    self.reservations = [[NSArray alloc] init];
+    PFQuery *query = [PFQuery queryWithClassName:@"Reservation"];
+    [query includeKey:@"dates"];
+    [query whereKey:@"itemId" equalTo:[self.listing objectId] ];
+    [query findObjectsInBackgroundWithBlock:^(NSArray * _Nullable objects, NSError * _Nullable error) {
+        if(error == nil){
+            self.reservations = objects;
+            [self.tableView reloadData];
+        }else{
+            NSLog(@"END: Error fetching reservation dates");
+        }
+    }];
+}
+
+- (IBAction)didTapBack:(id)sender {
+    [self dismissViewControllerAnimated:YES completion:nil];
+}
+
+- (nonnull UITableViewCell *)tableView:(nonnull UITableView *)tableView cellForRowAtIndexPath:(nonnull NSIndexPath *)indexPath {
```

Comment:
no need for the `nonnull` in the the implementations

Reply:
This is the default method header xcode gave me?